src/keycloak/chart/templates/istio-admin.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: keycloak-block-admin-access-from-public-gateway
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "keycloak.selectorLabels" . | nindent 6 }}
+  action: DENY
+  rules:
+  - to:
+    - operation:
+        paths: 
+          - "/admin*"
+          - "/realms/master*"
+          - "/metrics*"
+    when:
+    - key: source.namespace
+      notValues: ["istio-admin-gateway"]
```

I think this is equivalent, I tested it locally:
```suggestion
    from:
    - source:
      notNamespaces: ["istio-admin-gateway"]
```